src/components/Index/Environment.vue

```diff
@@ -33,8 +33,21 @@
       <small class="form-text text-muted">Set your timeout.</small>
     </div>
 
-    <div class="form-group" v-if="isEnabled(AllFeatures.pollingInterval)">
-      <label>Polling interval (milliseconds)</label>
+    <div class="form-group" v-if="isEnabled(AllFeatures.bucketing)">
+      <label>Flagship Mode</label>
+      <select
+        type="text"
+        class="form-control"
+        placeholder="Flagship Mode"
+        v-model="flagshipMode"
+      >
+        <option value="api">API</option>
+        <option value="bucketing">Bucketing</option>
+      </select>
+    </div>
+
+    <div v-if="flagshipMode == 'bucketing'" class="form-group">
```

Comment:
Could you add the condition isEnabled(AllFeatures.pollingInterval) here ?

Reply:
Yess of course

Reply:
Fixed